desktop/src/reports/dialogs/column-picker-dialog/column-picker-dialog.component.scss

```
@@ -0,0 +1,260 @@
+    padding: 8px 8px 8px 12px;
+    background: #f8fafc;
+  }
+
+  &__expr-text {
+    flex: 1;
+    min-width: 0;
+    align-self: center;
+    font-family: ui-monospace, Menlo, monospace;
+    font-size: 13.5px;
+    color: #1e293b;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)